labs/firepower-restapi-101/2.md

```diff
@@ -4,29 +4,29 @@ The API Explorer provides a limited interface for the REST API as well as giving
 
 The API Explorer resides on the FMC, and can be accessed via the Firepower Management Center at: https://yourownfmc/api/api-explorer/
 
-You can play live with API explorer on the FMC REST API sandbox
+You can play live with API Explorer on the FMC REST API sandbox:
 
 https://fmcrestapisandbox.cisco.com/api/api-explorer/
 
 This explorer provides the list of APIs available on the FMC, and their details.
 
 The first time you connect to the API Explorer you may receive an error that the connection is not secure due to an invalid certificate. You will need to add an exception in your browser to use the certificate and accept the connection.
-![Figure: FMC REST API explorer screen shot ](/posts/files/firepower-restapi-101/assets/images/fmcapiexp.PNG)
+![Figure: FMC REST API Explorer screen shot ](/posts/files/firepower-restapi-101/assets/images/fmcapiexp.PNG)
 ## Authentication to the API Explorer
 The REST API relies on the same authentication as the FMC UI. Each function in the REST API maps to permissions in the FMC.
 
-You can log into the API Explorer using any account on the FMC, but you will only be able to perform the functions for which the account has permissions. For more information on setting permissions for user roles, see the FMC Configuration Guide.
+You can log into the API Explorer using any account on the FMC, but you will only be able to perform the functions for which the account has permissions. For more information on setting permissions for user roles, see the [Firepower Management Center Configuration Guide](http://www.cisco.com/c/en/us/td/docs/security/firepower/610/configuration/guide/fpmc-config-guide-v61/logging_into_firepower_system.html).
 
-FMC REST API uses token based authentication. To invoke a REST call on FMC, you need to use a valid access token. On every REST call to access resource URLs you need to add a header with "key" set to “X-auth-access-token” and “value” containing a valid access token . Api explorer also follows this, but it is automated by Explorer UI. You can learn more in firepower-restapi-102 lab.
+FMC REST API uses token-based authentication. To invoke a REST call on FMC, you need to use a valid access token. On every REST call to access resource URLs you need to add a header with "key" set to “X-auth-access-token” and “value” containing a valid access token . Api Explorer also follows this, but it is automated by Explorer UI. You can learn more in [Firepower REST API 102 lab](https://learninglabs.cisco.com/lab/firepower-restapi-102/step/1).
 
```

Comment:
Good, I do see the requested change to the URL, now also look for an extra space after token and before the period for the third-to-last sentence, plus Api Explorer should be API Explorer.